monkey/infection_monkey/monkey.py

```diff
@@ -164,6 +172,24 @@ def _register_agent(self):
         )
         self._island_api_client.register_agent(agent_registration_data)
 
+    @staticmethod
+    def _get_os() -> Optional[OperatingSystem]:
+        system = platform.system()
+        if system == "Windows":
+            return OperatingSystem.WINDOWS
+        if system == "Linux":
+            return OperatingSystem.LINUX
+        return None
```

Comment:
I feel like this should never return `None`. I know of no scenario where the agent binary would successfully run but `platform.system()` would return something other than "Windows" or "Linux".

Reply:
Also, I think it would be better to move this kind of functions to some utils or better yet some `information_discovery` module.